### DISCUSSION TITLE
USB audio output?

### DISCUSSION THREAD

**Discussion by mrdangran** [dtcooper/raspotify#643] [Q&A]
HI is there a setting I can do on the Conf file to output audio to a USB sound device?  I ran "librespot -d ?" at the command line and got back this message on my usb card:

ALSA lib pcm_dmix.c:1075:(snd_pcm_dmix_open) unable to open slave
	Device:

		iec958:CARD=Device,DEV=0

	Description:

		USB PnP Sound Device, USB Audio
		IEC958 (S/PDIF) Digital Audio Output

	Supported Format & Sample Rate Combinations:

		Format: S16   Sample Rate(s): 44.1kHz, 48kHz

	--------------------------------------------------------------------

	Device:

		dmix:CARD=Device,DEV=0

	Description:

		USB PnP Sound Device, USB Audio
		Direct sample mixing device

	Supported Format & Sample Rate Combinations:

		Format: S16   Sample Rate(s): 48kHz

	--------------------------------------------------------------------

Thank you!!
Simon

**Comment by JasonLG1979** [comment]
Yes, it's right there in the config.

**Reply by mrdangran** [reply]
Sorry I'm a total newbie.  So I tried setting LIBRESPOT_DEVICE="audiodongle" but it's not working.  I can only get audio off of the headphone jack.  Is that the setting you are talking about ? Or do I need to actually put in something from the output I get from Librespot --device ?  like "iec958:CARD=Device,DEV=0"  ?  I  guess I can try different combos of what I see from the output.   Thanks
